modules/system/assets/ui/js/jquery.debouncedresize.js.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * debouncedresize: special jQuery event that happens once after a window resize
```

Comment:
@ayumihamsaki  double `.js.js` in the filename - `jquery.debouncedresize.js.js`

Reply:
thanks